documentation/projects/proposals/nuxt_3_migration/20230608-implementation_plan_nuxt_3_migration.md

```diff
@@ -0,0 +1,338 @@
+| `"@pinia/nuxt"`                    | Update to current version that supports Nuxt 3                                                          |                                                                                                                                                          |
+| `"@nuxtjs/proxy"`                  | Replace with Nitro routeRules or use `proxyRequest`                                                     | https://stackoverflow.com/questions/76120894/how-do-i-set-proxy-with-baseurl-with-nuxt3-and-nitro-config, https://github.com/nuxt/nuxt/discussions/15907 |
+| `"@nuxtjs/redirect-module"`        | Define Nitro routeRules in `nuxt.config`                                                                | https://nitro.unjs.io/config/#routerules                                                                                                                 |
+| `"@nuxtjs/sentry"`                 | No official support                                                                                     | Workaround in https://github.com/nuxt-community/sentry-module/issues/530                                                                                 |
+| `"vue-plausible"`                  | ~~Update to current version that supports Nuxt 3 or~~ Move to https://github.com/nuxt-modules/plausible | Note: enabling outbound tracking breaks links with target blank in `vue-plausible`                                                                       |
+| `"@nuxtjs/sitemap"`                | Replace with `nuxt-simple-sitemap`                                                                      | https://github.com/harlan-zw/nuxt-simple-sitemap                                                                                                         |
+
+We should probably add [`nuxt-vitest`](https://github.com/danielroe/nuxt-vitest)
+for testing in Nuxt. The package is under active development, so we should pin
+to the patch version
+
+## Alternatives
+
+<!-- Describe any alternatives considered and why they were not chosen or recommended. -->
+
+Migrate to a non-Nuxt SSR solution for Vue. This would require significant
+changes to code for an unclear benefit.
+
+## Design
+
+<!-- Note any design requirements for this plan. -->
+
+The designs should stay exactly the same. Hopefully, all of the Playwright
+visual regression tests pass after this migration.
+
+## Parallelizable streams
+
+<!-- What, if any, work within this plan can be parallelized? -->
+
+The Nuxt 2 changes can be worked on immediately. The changes to Nuxt 3 branch
+can be parallellized after the initial PR.
+
+## Blockers
+
+<!-- What hard blockers exist which might prevent further work on this project? -->
```

Comment:
Team capacity may be a legitimate concern here. At the very least, we would have to pull maintainers from other parts of the project if we plan to do this in Q3-Q4 of this year.